doc/source/drivers/vector/openfilegdb.rst

```diff
@@ -137,6 +137,14 @@ Layer Creation Options
 -  **CONFIGURATION_KEYWORD**\ =DEFAULTS/MAX_FILE_SIZE_4GB/MAX_FILE_SIZE_256TB:
    Customize how data is stored. By default text in UTF-8 and data up to 1TB
 
+-  **CREATE_SHAPE_AREA_AND_LENGTH_FIELDS**\ =YES/NO.
+   Defaults to NO (through CreateLayer() API). When this option is set,
+   a Shape_Area and Shape_Length special fields will be created for polygonal
+   layers (Shape_Length only for linear layers). When using ogr2ogr with a
```

Comment:
I think it's worth noting here something like "These fields will automatically be populated with the feature's area or length whenever a new feature is added to the dataset or an existing feature is amended."

Reply:
added